### PR TITLE
Fix typo and remove extra button

### PR DESCRIPTION
**Pull request by rszper**
Fix typo and remove extra button in the MLTransform_basic notebook.

------------------------

Thank you for your contribution! Follow this checklist to help us incorporate your contribution quickly and easily:

 - [ ] Mention the appropriate issue in your description (for example: `addresses #123`), if applicable. This will automatically add a link to the pull request in the issue. If you would like the issue to automatically close on merging the pull request, comment `fixes #<ISSUE NUMBER>` instead.
 - [ ] Update `CHANGES.md` with noteworthy changes.
 - [ ] If this contribution is large, please file an Apache [Individual Contributor License Agreement](https://www.apache.org/licenses/icla.pdf).

See the [Contributor Guide](https://beam.apache.org/contribute) for more tips on [how to make review process smoother](https://github.com/apache/beam/blob/master/CONTRIBUTING.md#make-the-reviewers-job-easier).

To check the build health, please visit [https://github.com/apache/beam/blob/master/.test-infra/BUILD_STATUS.md](https://github.com/apache/beam/blob/master/.test-infra/BUILD_STATUS.md)

GitHub Actions Tests Status (on master branch)
------------------------------------------------------------------------------------------------
[Build python source distribution and wheels](https://github.com/apache/beam/actions?query=workflow%3A%22Build+python+source+distribution+and+wheels%22+branch%3Amaster+event%3Aschedule)
[Python tests](https://github.com/apache/beam/actions?query=workflow%3A%22Python+Tests%22+branch%3Amaster+event%3Aschedule)
[Java tests](https://github.com/apache/beam/actions?query=workflow%3A%22Java+Tests%22+branch%3Amaster+event%3Aschedule)
[Go tests](https://github.com/apache/beam/actions?query=workflow%3A%22Go+tests%22+branch%3Amaster+event%3Aschedule)

See [CI.md](https://github.com/apache/beam/blob/master/CI.md) for more information about GitHub Actions CI or the [workflows README](https://github.com/apache/beam/blob/master/.github/workflows/README.md) to see a list of phrases to trigger workflows.

### CODE REVIEW

**Comment by rszper** [comment]
R: @damccorm

**Review by damccorm — COMMENTED** [review]
Thanks!

File: examples/notebooks/beam-ml/mltransform_basic.ipynb [hunk 1]
```diff
@@ -6,9 +6,6 @@
         "id": "view-in-github",
         "colab_type": "text"
       },
-      "source": [
-        "<a href=\"https://colab.research.google.com/github/AnandInguva/beam/blob/mltransform_notebook/examples/notebooks/beam-ml/mltransform_basic.ipynb\" target=\"_parent\"><img src=\"https://colab.research.google.com/assets/colab-badge.svg\" alt=\"Open In Colab\"/></a>"
-      ]
     },
```
Comment (damccorm) [inline examples/notebooks/beam-ml/mltransform_basic.ipynb:9]: Oh, I think we need to remove the full cell (from line 3 to this line). That should also fix the broken formatting - https://github.com/apache/beam/blob/2a7c9ff27fdbe2c6ed98945dd3f13db6b59f44d1/examples/notebooks/beam-ml/mltransform_basic.ipynb

Comment (rszper) [reply examples/notebooks/beam-ml/mltransform_basic.ipynb]: Ah, okay, I wondered about that. I removed the full cell.

**Review by damccorm — APPROVED** [review]
Looks good to me, thanks!

**Pull request merged**